draft-rescorla-tls-rfc8446-bis.md

```diff
@@ -1632,7 +1633,7 @@ Here:
 - "extension_data" contains information specific to the particular
   extension type.
 
-The list of extension types is maintained by IANA as described in
+IANA maintains the extension type list as described in
```

Comment:
Let's revert this. I agree the phrasing is clumsy but I'd prefer not to make too any unneeded text changes.